lang/test/org/partiql/lang/syntax/SqlParserMatchTest.kt

```diff
@@ -0,0 +1,516 @@
+package org.partiql.lang.syntax
+
+import com.amazon.ionelement.api.ionBool
+import com.amazon.ionelement.api.ionInt
+import com.amazon.ionelement.api.ionString
+import org.junit.Ignore
+import org.junit.Test
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.domains.id
+import org.partiql.pig.runtime.asPrimitive
+
+class SqlParserMatchTest : SqlParserTestBase() {
+    @Test
+    fun allNodesNoLabel() = assertExpressionNoRoundTrip(
+        "SELECT 1 FROM my_graph MATCH ()"
+    ) {
+        select(
+            project = projectList(projectExpr(lit(ionInt(1)))),
+            from = graphMatch(
+                expr = id("my_graph"),
+                graphExpr = graphMatchExpr(
+                    patterns = listOf(
+                        graphMatchPattern(
+                            parts = listOf(
+                                PartiqlAst.GraphMatchPatternPart.Node(
```

Comment:
nit: could use `node_` rather than calling the `PartiqlAst.GraphMatchPatternPart.Node` constructor. Similarly for the other lines

```suggestion
                                node_(
```